src/metadataGeneration/metadataGenerator.ts

```diff
@@ -87,4 +87,4 @@ export class MetadataGenerator {
       .filter(generator => generator.IsValid())
       .map(generator => generator.Generate());
   }
-}
```

Comment:
Formatting

Reply:
Seems like you avoided all checks to be able to push this.
You can still get the formatting to not be off by running `yarn run prettier **/*.ts --write`